Update upgrades.md (#747)
rancher/rancher-docs: docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/upgrades.md

:::

:::tip 

> Your deployment name may vary; for example, with the AWS Marketplace offering of Rancher Setup, the deployment name is 'rancher-stable'. 

Reviewer: ```suggestion
Your deployment name may vary; for example, if you're deploying Rancher through the AWS Marketplace, the deployment name is 'rancher-stable'. 
```
